azure-armrest.gemspec

```diff
@@ -19,7 +19,7 @@ behind the scenes.
 
   spec.add_dependency 'json', '~> 2'
   spec.add_dependency 'rest-client', '~> 2.0.0'
-  spec.add_dependency 'memoist', '~> 0.15.0'
+  spec.add_dependency 'memoist', '>= 0.15.0'
```

Comment:
Would `~> 0.15` be better so as not to bring in a 1.0 it that ever happens?

Reply:
good idea